schemas/manifest.fbs

```diff
@@ -95,6 +95,9 @@ table ManifestV1 {
     // needs this to determine whether it's safe to drop duplicate key writes. If a recent snapshot
     // still references an older version of a key, it should not be dropped.
     recent_snapshot_min_seq: ulong;
+
+    // a mapping from sequence number to the corresponding system clock timestamp
+    seq_tracker: SequenceTracker;
```

Comment:
WDYT about `seq_system_ts_map: [SequenceTrackerTier]` and adding a `level` or `tier` field to the tier data model? I kinda feel like the `SequenceTracker` layer adds complexity and reflects the in-memory code/class/struct model rather than the data model. I don't feel strongly about this, but it seems simpler. Do you anticipate adding more fields to sequence tracker?

Reply:
let me play around with that, I wasn't sure there wasn't going to be other metadata in the top-level but I imagine worst case we can duplicate that metadata into the tier data model.

Reply:
Done, I think this makes sense to me - ultimately I think have a set of tracker tiers makes sense also from a mental model perspective. There's also no reason they have to be non-overlapping or related in any way, we could decide to change that for various reasons and just enforce how we merge them in the code.